README.md

```diff
@@ -242,7 +242,6 @@
   |> String.slice(1, 3)
   ```
 
-  <a name="needless-pipeline"></a>
```

Comment:
heh, I agree that this rule is important
I was think that it's mistake:
> As I can see all other rules have links only at end of first paragraph for rule (list element in list for header level 3 - h3 - element)

When I parsed this file I was just surprised to have html link inside my doc.
Is it a rule of styling your list? If not I should revert this change. Otherwise how about wrap it into separate list element?
It's not really important for me, but I asked to be sure that this is not formatting mistake. In referenced issue I added (optional) question mark, because I was not sure if this single case is not a mistake.